crash-lib/src/jvmMain/kotlin/co/electriccoin/zcash/crash/ReportedExceptionExt.kt

```diff
@@ -0,0 +1,23 @@
+package co.electriccoin.zcash.crash
+
+import kotlinx.datetime.Instant
+import java.io.File
+
+fun ReportedException.Companion.new(file: File): ReportedException? {
+    // Exclude temp files
+    if (file.extension != "txt") {
+        return null
+    }
+
+    val name: String = file.nameWithoutExtension
+    val splitName = name.split(ExceptionPath.SEPARATOR)
+
+    val epochSeconds = splitName.firstOrNull()?.toLongOrNull()
+    val classNameString = splitName.lastOrNull()
+
+    if (null != epochSeconds && null != classNameString) {
```

Comment:
I need a test confirming the filename generation and parsing are always in sync.